src/Sentry.Unity.Editor/ConfigurationWindow/EnrichmentTab.cs

```diff
@@ -72,6 +72,10 @@ internal static void Display(ScriptableSentryUnityOptions options)
                 new GUIContent("Report Assemblies Mode", "Whether or not to include referenced assemblies " +
                                                          "Version or InformationalVersion in each event sent to sentry."),
                 options.ReportAssembliesMode);
+
+            options.AttachScreenshot = EditorGUILayout.Toggle(
+                new GUIContent("Attach Screnshot", "Try to attach current screenshot for error events"),
```

Comment:
```suggestion
                new GUIContent("Attach Screnshot", "Try to attach current screenshot for error events.\n" +
                  "A screenshot might not be able to be attached, for example when the error happens on a background thread."),
```

Reply:
btw I've removed "error" because with the current implementation, it's attached to messages as well.